examples/learning/helloworld.html

```diff
@@ -29,42 +30,42 @@ <h1>'Hello, world!' example</h1>
   // Asynchronous download PDF
   //
   var loadingTask = pdfjsLib.getDocument(url);
-  loadingTask.promise.then(function(pdf) {
-    //
-    // Fetch the first page
-    //
-    pdf.getPage(1).then(function(page) {
-      var scale = 1.5;
-      var viewport = page.getViewport({ scale: scale, });
-      // Support HiDPI-screens.
-      var outputScale = window.devicePixelRatio || 1;
+  (async () => {
+  const pdf = await loadingTask;
+  //
+  // Fetch the first page
+  //
+  const page = await pdf.getPage(1);
+  var scale = 1.5;
+  var viewport = page.getViewport({ scale });
+  // Support HiDPI-screens.
+  var outputScale = window.devicePixelRatio || 1;
+
+  //
+  // Prepare canvas using PDF page dimensions
+  //
+  var canvas = document.getElementById("the-canvas");
+  var context = canvas.getContext("2d");
 
-      //
-      // Prepare canvas using PDF page dimensions
-      //
-      var canvas = document.getElementById('the-canvas');
-      var context = canvas.getContext('2d');
+  canvas.width = Math.floor(viewport.width * outputScale);
+  canvas.height = Math.floor(viewport.height * outputScale);
+  canvas.style.width = Math.floor(viewport.width) + "px";
+  canvas.style.height = Math.floor(viewport.height) + "px";
 
-      canvas.width = Math.floor(viewport.width * outputScale);
-      canvas.height = Math.floor(viewport.height * outputScale);
-      canvas.style.width = Math.floor(viewport.width) + "px";
-      canvas.style.height =  Math.floor(viewport.height) + "px";
+  var transform =
+    outputScale !== 1 ? [outputScale, 0, 0, outputScale, 0, 0] : null;
 
-      var transform = outputScale !== 1
-        ? [outputScale, 0, 0, outputScale, 0, 0]
-        : null;
+  //
+  // Render PDF page into canvas context
+  //
+  var renderContext = {
+    canvasContext: context,
+    transform,
+    viewport,
+  };
+  page.render(renderContext);
```

Comment:
Note that this *entire* block of code should be indented with *two* more spaces.